### DISCUSSION TITLE
MISTRAL not appearing anymore

### DISCUSSION THREAD

**Discussion by a7t0fwa7** [danny-avila/LibreChat#1853] [Troubleshooting]
### What happened?

Hi Mistral AI is not appearing anymore in my options since I have updated the repo. Is this intentional? I didn't have issues with this before

### Steps to Reproduce

In the .env file MISTRAL seems to have been removed will this be re-installed or do we need to add this manually

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Not intentional, not a bug. I'm not sure where you had MISTRAL configured in the .env file? Anyway, updates can't change your personal .env file.

I update a remote VPS and have had no issues with my config using the example from `librechat.yaml.example`:
![image](https://github.com/danny-avila/LibreChat/assets/110412045/698927c2-be61-4619-a5c1-c019c55ef66f)


Make sure you have your `librechat.yaml` file configured: https://docs.librechat.ai/install/configuration/custom_config.html

If you are using docker, see the docker setup: https://docs.librechat.ai/install/configuration/custom_config.html#docker-setup
